src/Stratis.Features.FederatedPeg.Tests/Chain_With_NetworkExtension_Shall.cs

```diff
@@ -11,7 +10,7 @@ namespace Stratis.Features.FederatedPeg.Tests
     [Collection("FederatedPegTests")]
     public class Chain_With_NetworkExtension_Shall
     {
-        [Fact(Skip = TestingValues.SkipTests)]
+        [Fact]
```

Comment:
Ah @codingupastorm, whilst you are here, can you perhaps camel case these test names?

Reply:
Oop, these tests are going to get removed in one PR or another so I'll kill them here